toqito/matrix_ops/tests/test_tensor_comb.py

```diff
@@ -1,50 +1,68 @@
-"""Test tensor_comb."""
-
+"""Test tensor_comb for different cases."""
 import numpy as np
 import pytest
 
-from toqito.matrix_ops import tensor_comb
-from toqito.states import basis
-
-e_0, e_1 = basis(2, 0), basis(2, 1)
+from toqito.matrix_ops import tensor_comb, to_density_matrix
 
 
 @pytest.mark.parametrize(
-    "states, k, expected",
+    "k, states, mode, density_matrix, expected_keys, expected_error",
```

Comment:
I don't understand why you removed these test cases. Couldn't you just add another argument for the mode and include the additional test cases here?